### ISSUE TITLE
InlinePanel with min_num = max_num = 1 in tab other than 'content_panel' doesn't set the "add" button as disabled 

### ISSUE THREAD

**Issue by pyMan** [wagtail/wagtail#3316]
Really a little minor ui-bug

```
InlinePanel('related_links', label = "related links", min_num = 1, max_num = 1)
```

If the InlinePanel is used in the content_panel (or maybe just the first tab) the 'add' button is disabled. If the same InlinePanel is used in another tab, the 'add' button is not disabled.

Anyway the backend does its correct validation about the number of posted elements.

Things instead work correctly if max_num has a greater value and you try adding elements with the button (it becomes disabled when max_num is reached), but still the button is not disabled if the number of elements is already the same of max_num when the admin page is displayed.

Wagtail 1.7

**Comment by jjanssen** [comment]
Related #2687 Which has the same origin.

**Comment by gasman** [comment]
Fixed in #3679

**Comment by gasman** [comment]
Fixed in #3679
